src/public/server.js

```diff
@@ -70,33 +78,33 @@ ServerController.prototype = {
 /* -------- End ServerController Class -------- */
 
 
-/* -------- Games Sessions -------- */
+/* -------- controllers Sessions -------- */
 
 /**
- * Games sessions
- * @param {array} Games
+ * controllers sessions
+ * @param {array} controllers
  */
-var games = [];
+var controllers = [];
 
 /**
  * List Rooms for UI and selection
  */
 function listRooms() {
-  return games.map(function (game) {
-    return game.toJSON();
+  return controllers.map(function (controller) {
+    return controller.game.toJSON();
   });
 }
 
 function findByDungeonId(id) {
-  for (var i = 0; i < games.length; i++) {
-    for (var x = 0; x < games[i].dungeons.length; x++) {
-      if (games[i].dungeons[x].id === id) return games[i];
+  for (var i = 0; i < controllers.length; i++) {
+    for (var x = 0; x < controllers[i].game.dungeons.length; x++) {
+      if (controllers[i].game.dungeons[x].id === id) return controllers[i];
     }
   }
```

Comment:
For me there is a problem with dungeons stuff. I'll try to understand it better.